plutus-core/untyped-plutus-core/test/Transform/inlinePure4.uplc.golden

```diff
@@ -1,9 +1 @@
-[
-  (lam a_4 (lam b_5 a_4))
-  (force
-    [
-      (lam x_6 (delay (lam y_7 [ x_6 x_6 ])))
-      (delay [ (error) (con integer 1) ])
-    ]
-  )
-]
\ No newline at end of file
+(\a_4 b_5 -> a_4) (force ((\x_6 -> delay (\y_7 -> x_6 x_6)) (delay (error 1))))
```

Comment:
Hm, why is it a single line? I thought it would render as two. I'll look into the UPLC pretty-printer.